tests/test_client.py

```diff
@@ -1,11 +1,23 @@
 from faunadb.client import FaunaClient
 from faunadb.errors import UnexpectedError
+from faunadb import query
 from tests.helpers import FaunaTestCase
 
 class ClientTest(FaunaTestCase):
 
   def test_ping(self):
+    old_time = self.client.last_txn_time.get_time()
     self.assertEqual(self.client.ping("node"), "Scope node is OK")
+    new_time = self.client.last_txn_time.get_time()
+
+    self.assertEqual(old_time, new_time) # client.ping should not update last-txn-time
+
+  def test_client(self):
+    old_time = self.client.last_txn_time.get_time()
+    self.client.query(query.let({"x": 1}, query.var("x")))
+    new_time = self.client.last_txn_time.get_time()
+
+    self.assertTrue(old_time < new_time) # client.query should update last-txn-time
```

Comment:
if `last_txn_time.get_time()` returns `None` if not initialized, what are the changes `old_time` will be `None`

Reply:
Not sure I follow?

If you're wondering about the comparison, seems like `None` is less than everything else.

```
>>> None == None
True
>>> None < 5
True
```

Reply:
that was my concern, if `None` is comparable with integer

Reply:
Got it.  Do you think it's Too Clever and should the default ts be just `0` or something?  I went back and forth on this.

Reply:
it's fine to me

Reply:
if you set the default to 0, on `get_header` you'll have to compare with 0 or return `{ "X-Last-Txn-Time" : 0 }` for the first time, is it a problem?

Reply:
Yeah, trying to get out of the compare-with-in-theory-valid-timestamp situation was what made me converge on what passes for an Option type here :)